app/src/main/java/com/google/developer/udacityalumni/fragment/ArticleFragment.java

```diff
@@ -99,6 +100,12 @@ public void onShareClick(String title) {
 //                ie: https://udacity-alumni-client.herokuapp.com/articles/medically-necessary-utilization-review-evidence-of-insurability
 //                TODO: Share DYNAMIC link to the article  (the 'url' above is a link to the article on the web app)
 //               Will implement later
+                    Intent i = new Intent(Intent.ACTION_SEND);
+                    i.setType("text/plain");
+                    i.putExtra(Intent.EXTRA_SUBJECT,title);
+                    i.putExtra(Intent.EXTRA_TEXT,R.string.share+url);
```

Comment:
This should be `getString(R.string.share)`, otherwise you will print the numerical id of `R.string.share` and not the string resource.

Also, instead of concatenating strings with a `+` symbol, it would be better to use a placeholder `xml` resource, i.e.:
 
    i.putExtra(Intent.EXTRA_TEXT, getString(R.string.share, url));
    //...
    <string name="share">"\nLet me recommend this article:\n\n"<xliff:g id="url">%s</xliff:g></string>